### ISSUE TITLE
Fix Windows builds

### ISSUE THREAD

**Issue by mrakitin** [nsls-ii-forge/pycentroids-feedstock#2]
The log of the failed Windows build can be found [here](https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=2670&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=1183ba29-a0b5-5324-8463-2a49ace9e213). Here is the specific issue:
```
  -- Building for: Visual Studio 15 2017 Win64
  CMake Error at CMakeLists.txt:2 (project):
    Generator

      Visual Studio 15 2017 Win64

    does not support platform specification, but platform

      x64

    was specified.


  -- Configuring incomplete, errors occurred!
```
@stuwilkins, any thoughts about it?

**Comment by mrakitin** [comment]
Building Windows packages was excluded in https://github.com/nsls-ii-forge/pycentroids-feedstock/pull/1.

**Comment by stuwilkins** [comment]
No.. but googling found this:

https://stackoverflow.com/questions/48624415/how-to-fix-cmake-error-in-cmakelists-txt-generator-nmake-makefiles-does-not-sup

**Comment by mrakitin** [comment]
OK, thanks, that's what I found yesterday too. I will see how it can be done with the existing infrastructure. Maybe a different combination of dependencies will be needed for Windows.

**Comment by mrakitin** [comment]
@mariusvniekerk, @isuruf, @cj-wright, could you please let us know if you observed the issue described above and know how to fix it? Is it something which can be solved upstream (e.g., in `conda-forge-pinning`)?

**Comment by isuruf** [comment]
Why do you think this is a `conda-forge` issue? Have you asked the upstream author why they added https://github.com/NSLS-II/centroids/blob/master/setup.py#L63 ?

**Comment by mrakitin** [comment]
Thanks @isuruf, we will discuss that flag. I am not claiming that's a conda-forge issue, just wanted to get your feedback as you may have seen such issues before. Thanks for looking into that!

**Comment by stuwilkins** [comment]
@mrakitin do you want to try removing that flag, it was just copied from the pybind example

**Comment by mrakitin** [comment]
Yes, we can try and see how it goes. I assume for Windows builds we need commands like `<executable> /<option> /<another option> ...`.
